docs/r_sig/23_11_06_renv/index.qmd

```diff
@@ -23,13 +23,17 @@ renv::init()
 
 This mainly will do two things:
 
-1.  Create a project specific package library, which contains all the packages currently used by the project. This means different projects can use different package versions.
+1.  Create a project specific package library in the folder `renv` in your working directory, which contains all the packages currently used by the project. This means different projects can use different package versions, and even different R-Versions.
```

Comment:
maybe add the info somewhere, that the renv folder doesn't get uploaded on github unless you use snapshot (when I understood that right). because that confused me somewhat.